package.json

```diff
@@ -10,13 +10,12 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@sentry/react-native": "2.3.0",
-    "react": "16.13.1",
-    "react-native": "0.63.2",
-
     "@react-native-community/masked-view": "^0.1.10",
     "@react-navigation/native": "^5.7.3",
     "@react-navigation/stack": "^5.9.0",
+    "@sentry/react-native": "2.4.0",
```

Comment:
These 3 are huge Adam nice work, please put this in your PR description as well as the README

Reply:
Thanks! Yup, done.